404.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
```

Comment:
혹시나 이 파일이 사용되는 파일이라면, 언어셋을 지정해주면 더 좋을 것 같아요!
지정 안해두면 크롬에서는 번역 할거냐는 창이 자꾸 떠서 불편하더라구요 ㅋㅋㅋㅋㅋㅋ
```suggestion
<html lang="ko">
```

Reply:
엄멈머 좋습니다!!!